src/locales/fr.json

```diff
@@ -22,13 +22,27 @@
       "WALKING": "Marche",
       "BUS": "Bus"
     },
+    "purposes": {
+      "HOME": "Domicile",
+      "WORK": "Travail",
+      "SCHOOL": "École",
+      "SHOPPING": "Achats",
+      "MEAL": "Repas",
+      "PERSONAL_MED": "Personnel",
+      "EXERCISE": "Sport",
+      "ENTERTAINMENT": "Loisir",
+      "PICK_DROP": "Dépose",
+      "OTHER_PURPOSE": "Indéfini"
+    },
     "duration": "Durée",
     "distance": "Distance",
     "calories": "Calories",
     "co2": "CO2"
   },
   "loadMore": "Charger plus de trajets",
   "tripEdit": {
-    "selectMode": "Modifier le mode de transport"
-  }
+    "selectMode": "Modifier le mode de transport",
+    "selectPurpose": "Modifier le motif du déplacement"
+  },
+  "purpose": "Motif de déplacement"
```

Comment:
In line 45 you used "motif **du** déplacement", and here you use "motif **de** déplacement". Is that wanted?